java/src/main/java/cucumber/api/java8/StepdefBody.java

```diff
@@ -2,42 +2,42 @@
 
 public interface StepdefBody {
     public static interface A0 extends StepdefBody {
-        void accept();
+        void accept() throws Exception;
```

Comment:
[Utils#invoke](https://github.com/cucumber/cucumber-jvm/blob/master/core/src/main/java/cucumber/runtime/Utils.java#L45) accepts any Throwable. To remain consistent with the regular step defintions all methods should declare throw Throwable.